app/models/notification.rb

```diff
@@ -0,0 +1,41 @@
+class Notification < ActiveRecord::Base
```

Comment:
Ok, now it makes more sense. And I guess that on the front end there is going to be link to the notifications then? I thought it was just a number to show. Let me have a look again.